src/main/java/bio/terra/cli/command/groups/AddUser.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cli.command.groups;
+
+import bio.terra.cli.auth.AuthenticationManager;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.utils.SamService;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra groups add-user" command. */
+@Command(name = "add-user", description = "Add a user to a group.")
+public class AddUser implements Callable<Integer> {
+  @CommandLine.Parameters(index = "0", description = "The email of the user.")
+  private String user;
+
+  @CommandLine.Option(names = "--group", required = true, description = "The name of the group")
+  private String group;
+
+  @CommandLine.Option(
+      names = "--policy",
+      required = true,
+      description = "The name of the policy: ${COMPLETION-CANDIDATES}")
+  private SamService.GroupPolicy policy;
+
+  @Override
+  public Integer call() {
+    GlobalContext globalContext = GlobalContext.readFromFile();
+    WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
+
+    new AuthenticationManager(globalContext, workspaceContext).loginTerraUser();
```

Comment:
doesn't need to be addressed in this PR: Do I need to have a workspace context to use the authentication manager? It'd probably be nice to be able to run the group/spend commands outside of a workspace context since those commands are independent of having a workspace. While the majority of commands are going to be workspace related, I think we will have a sizable fraction of commands that are above the level of workspaces.

Reply:
No, you don't need one. And you can run the group/spend commands without a workspace context.

That workspace context argument in the AuthManager constructor should go away once I get to the multiple identity contexts ticket. The pet SA key file fetch should not be happening during login anyway, and that's the only reason we need the workspace context in that class.